content/applications/sales/subscriptions/renewals.rst

```diff
@@ -1,80 +1,70 @@
-====================
-Renew a subscription
-====================
+===================
+Renew subscriptions
+===================
 
-The key feature of a subscription business model is the recurring nature of payments. In this model,
-customers pay a recurring amount in exchange for access to a product or a service.
+The foundation of any subscription business model is recurring payments. That's when customers
+reliably pay a regular amount at specific intervals, in exchange for access to a subscription
+product or service.
 
-.. raw:: html
+The renewal of a subscription is the process followed by customers when they willingly choose to
+continue their participation (and payment) for a subscription product or service.
 
-   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>The renewal of a
-   subscription is the process followed by each customer when willing to pursue a
-   subscription.</b> </div>
+Subscribers experience the renewal process at different intervals -- weekly, monthly, annually, etc.
+-- depending on the duration of the agreed-upon contract.
 
-Each subscriber experiences this renewal process monthly, annually, or sometimes more, depending on
-the duration of the contract. Most subscription companies choose to automate their renewal processes
-but, in some cases, manual subscription renewals are still the preferred option.
+Most companies that offer subscriptions prefer to automate the renewals process for customers, but,
+in some cases, manual subscription renewals are still used.
 
-With **Odoo Subscriptions**, you can have all your subscriptions in one application, suggest an
-automatic subscription renewal to your customers (as well as a manual one) and, finally, filter all
-your subscriptions and easily find those to renew (with the help of the tag *To renew*).
+With the Odoo *Subscriptions* application, a company can accurately manage all subscription renewals
+in a single platform. Users can even filter all current subscriptions to quickly locate customers
+who need to renew.
 
-Renew your first subscription
-=============================
+Subscription renewals
+=====================
 
-Before renewing a subscription, be sure to check out our documentation on how to
-:doc:`Create a quotation <../subscriptions>` using subscription
-products. Indeed, once confirmed, a quotation becomes a sales order and a new subscription is
-automatically created. Therefore, this subscription has the status *In progress*. From there, you
-have the possibility to renew the subscription. In the Other Info tab, underneath the To Renew
-section, you can activate the *To renew* option. When activated, a yellow tag automatically appears
-in the upper right corner of the subscription.
+In order to renew a subscription, a quotation with a subscription product(s)/service(s) **must** be
```

Comment:
List the necessary requirement only:
- Only a singular product is required
- A subscription service is still a product (albeit a digital + recurring one)

```suggestion
In order to renew a subscription, a quotation with a subscription product **must** be
```